src/api.ts

```diff
@@ -45,7 +45,7 @@ export interface ErrorInfo {
 /** Information about a line where a prop was used */
 export interface LineInfo {
   propCode: string;
-  propValue: string | symbol;
+  propValue: string | symbol | undefined;
```

Comment:
Sorry, I just realized, this would a backwards incompatible change 😭 Can you remove the `| undefined` and go back to `formatPropValue(null)`? Then I'll merge the PR.

I want to change how `propValue` works eventually. I think a union type would be good eventually (but not in this PR).

```ts
export type PropValue =
  | { type: "none" }
  | { type: "expression" }
  | { type: "literal", value: number | string | boolean | undefined | null; stringValue: string };
```

But that would be best done in a new PR and it will need a major version bump.